src/ExerciseOOP/Book.java

```diff
@@ -48,4 +48,14 @@ public String getAvailabilityStatus() {
     public void setAvailabilityStatus(String availabilityStatus) {
         this.availabilityStatus = availabilityStatus;
     }
+
+    @Override
```

Comment:
Why do we have @Override annotation here?

Reply:
Because I use this method to print the object